regression-test/package.json

```diff
@@ -6,6 +6,8 @@
   "scripts": {
     "build:webpack4": "webpack",
     "build:esbuild": "node esbuild.cjs",
+    "build:next": "next build ./src",
+    "start:next": "next start ./src ",
```

Comment:
This has to be changed to `next dev ./src`, since we wan't to test the development build of the next.js app.